src/test/java/com/softserveinc/edu/ita/tests/OrderingTabSortingTest.java

```diff
@@ -0,0 +1,189 @@
+package com.softserveinc.edu.ita.tests;
+
+import com.softserveinc.edu.ita.dao_jdbc.domains.User;
+import com.softserveinc.edu.ita.dao_jdbc.domains.Orders;
+import com.softserveinc.edu.ita.dataproviders.DataProviders;
+import com.softserveinc.edu.ita.locators.HomePageLocators;
+import com.softserveinc.edu.ita.page_object.HomePage;
+import com.softserveinc.edu.ita.page_object.OrderingPage;
+import com.softserveinc.edu.ita.page_object.UserInfoPage;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * This class is used to test sorting actions in 'Ordering' table of 'Ordering' page.
+ */
+public class OrderingTabSortingTest extends TestRunner {
+    //Test sorting by Order Name
+    @Test(dataProvider = "getMerchandisers", dataProviderClass = DataProviders.class)
+    public void testOrderNameColumn(User user) {
+        HomePage homePage = new HomePage(driver);
+        UserInfoPage userInfoPage = homePage.logIn(user.getLogin(), user.getPassword());
+        OrderingPage orderingPage = userInfoPage.clickOrderingTab();
+
+        List<Orders> baseTable = orderingPage.getTable();
+        baseTable.sort(Comparator.comparing(Orders::getOrderName));
+
+        orderingPage.clickTableColumn(com.softserveinc.edu.ita.enums.Orders.ORDER_NAME.toString());
```

Comment:
once Orders is renamed, full name is not needed - com.softserveinc.edu.ita.enums.Orders.ORDER_NAME